docs/computing/connecting/index.md

```diff
@@ -0,0 +1,167 @@
+# Connecting to CSC supercomputers
+
+There are two main ways of connecting to CSC supercomputers. The traditional way to
+work on a supercomputer is [using an SSH client](#using-an-ssh-client). We
+also offer a [web interface](#using-the-web-interface) to our systems, which
+enables running both graphical applications and command-line shells.
+
+For instructions on connecting to the LUMI supercomputer, please see the
+[Get Started page in the LUMI user guide](https://docs.lumi-supercomputer.eu/firststeps/getstarted/).
+
+!!! note "Login node usage policy"
+
+    When you connect to a supercomputer using an SSH client or the
+    [*Login node shell*](../webinterface/shell.md) app, you are directed to a
+    login node.  Login nodes can be used for light pre- and postprocessing
+    tasks, compiling applications and moving data. **All other tasks are to be
+    done on compute nodes** using the
+    [batch job system](../running/getting-started.md). Processes not adhering
+    to these rules will be terminated without warning. Compute
+    nodes can also be [used interactively](../running/interactive-usage.md).
```

Comment:
Linking to usage-policy.md page here would be good (e.g. to clarify what is meant with "light"). The text could also be shorter (most of this is already said in the usage-policy page), but it is not a big deal. You can leave it as is if you think it is better to be more verbose.

Reply:
Good point. e60c0fe0